packages/material-ui/src/Tabs/TabScrollButton.js

```diff
@@ -28,7 +28,14 @@ const TabScrollButton = React.forwardRef(function TabScrollButton(props, ref) {
   }
 
   return (
-    <ButtonBase className={className} onClick={onClick} ref={ref} tabIndex={-1} {...other}>
+    <ButtonBase
+      component="div"
```

Comment:
What about using the native button and just aria-hidden it if it's actually not reachable/required for impaired users?

Reply:
It was my first tentative, didn't work.

Reply:
I'm trying something else.